src/load.jl

```diff
@@ -172,3 +172,68 @@ function load(fname; repair=true, layer=0, lenunit=nothing, numbertype=Float64,
   # perform repairs
   geotable |> pipeline
 end
+
+
+"""
+    GeoIO.loadvalues(fname; emptyonly=true, kwargs...)
+
+Load non-geographic information as a table from file `fname` stored in 
+.shp, .geojson, .parquet or other ArchGDAL-compatible format.
+
+This function does not process geographic information, allowing for 
+  loading of non-geographic data from large files.
+
+The return object is a Tables.jl - compatible table, and can be easily
+converted with e.g. `DataFrame()` from DataFrames.jl.
+
+## Options
+
+* `emptyonly`: load only rows with empty or missing geometries. Useful to inspect geometries that have failed to load (and thrown a warning) in `GeoIO.load()`.
+
+## See Also:
+
+[`GeoIO.load()`](@ref)
+
+## Examples
+
+```julia
+# load non-geographic data as Tables.jl table
+GeoIO.loadvalues("file.shp")
+# load only empty geometries
+GeoIO.loadvalues("file.shp", emptyonly=true)
+```
+"""
+function loadvalues(fname; emptyonly=false, kwargs...)
+
+  # GIS formats only
+  table = if endswith(fname, ".shp")
+    SHP.Table(fname; kwargs...)
+  elseif endswith(fname, ".geojson")
+    GJS.read(fname; numbertype=Float64, kwargs...) #numbertype not relevant here
+  elseif endswith(fname, ".parquet")
+    GPQ.read(fname; kwargs...)
+  else # fallback to GDAL
+    data = AG.read(fname; kwargs...)
+    AG.getlayer(data, layer)
+  end
```

Comment:
Let's create a separate `gistable` function in `src/utils.jl`  to avoid code repetition. It can be used in `GeoIO.load` and in `GeoIO.loadvalues`.